LocalMultiplayerAgent.UnitTest/ConfigurationTests.cs

```diff
@@ -250,5 +250,20 @@ public void EmptyNodePortInContainerModeShouldFail()
             new MultiplayerSettingsValidator(settings, _mockSystemOperations.Object).IsValid().Should().BeFalse();
 
         }
+
+        [TestMethod]
```

Comment:
can you add a couple more test cases? e.g. a test that succeeds and multiple variations of startgamecommand.